mappings/net/minecraft/world/World.mapping

```diff
@@ -11,7 +11,7 @@ CLASS none/aiw net/minecraft/world/World
 	FIELD B lootTableHandler Lnone/bbm;
 	FIELD C profiler Lnone/os;
 	FIELD D scoreboard Lnone/bdb;
-	FIELD E isRemote Z
+	FIELD E isProxy Z
```

Comment:
Client doesn't represent it properly though, the variables should be named in context to the object they are on, in this case world.

world.isClient being true for worlds representing a server, but false for the embedded server on the client would be just wrong.